UT4MasterServer.Web/src/pages/Profile/PlayerCard.vue

```diff
@@ -1,8 +1,81 @@
 <template>
-  <h1>Player</h1>
-  <h3>Coming Soon!</h3>
-  <!-- 
-    Placeholder for the playerCard UI 
-  https://www.epicgames.com/unrealtournament/en-US/playerCard?playerId=0b0f09b400854b9b98932dd9e5abe7c5
```

Comment:
Link in this comment could be left in, so that we don't forget about original link after shutdown.

Reply:
Agreed